### DISCUSSION TITLE
Function call completion

### DISCUSSION THREAD

**Discussion by Msadat97** [coder/code-server#4434] [Q&A]
Hi, 

I'm using code-server and python extension, and there is a feature that has frustrated me recently. Whenever I use autocompletion for typing a function name, it automatically types its arguments too. This feature is not present in the original vscode. Could you please tell me how to disable this type of autocompletion?

**Comment by jsjoeio** [comment]
Hmm...I'm not sure where this is coming from. Maybe there is a setting similar to [this one](https://github.com/Microsoft/vscode-go/issues/1022#issuecomment-306998128) that you can disable? This isn't something that code-server is specifically adding so it could be related to your extensions or your `settings.json`

**Reply by Msadat97** [reply]
I removed all extensions and settings but the same behavior still exists. I'm using code-server 3.12.

**Reply by lg-nineoneninethreenine** [reply]
Try checking your Settings GUI for autocomplete and find the one, it has to be in the settings file.

**Reply by Msadat97** [reply]
I typed autocomplete in the search menu and disabled every option but the problem still persists.

**Reply by jsjoeio** [reply]
Can you post your `settings.json`? I don't think it's called "autocomplete" btw. Probably something around suggestions

**Reply by Msadat97** [reply]
```
{
    "editor.fontSize": 15,
    "scm.inputFontSize": 15,
    "debug.console.fontSize": 15,
    "markdown.preview.fontSize": 15,
    "terminal.integrated.fontSize": 15,
    "workbench.colorTheme": "One Dark Pro Darker",
    "workbench.iconTheme": "material-icon-theme",
    "python.linting.enabled": false,
    "python.linting.flake8Enabled": false,
}
```

**Reply by jsjoeio** [reply]
I don't see anything there that looks fishy. I'm sorry, I'm not sure how I can help you :(

**Reply by Msadat97** [reply]
@jsjoeio I tried an older version of code-server with an older python extension, and this behavior does not exist. Do you think it's a new update to the python extension that does this?

**Reply by Msadat97** [reply]
I can confirm that this feature has been added in the python extension version v2021.10, and it's not present in v2021.9

**Reply by lg-nineoneninethreenine** [reply]
Can you try downgrading the extension by getting the 2021.09 VSIX, and loading it up onto code-server by the CLI?

**Reply by Msadat97** [reply]
I'm already using the older version. But it would be great to have a toggle for setting this feature off in the future updates of the extension.

**Reply by jsjoeio** [reply]
@Msadat97 great debugging!

> But it would be great to have a toggle for setting this feature off in the future updates of the extension.

Maybe you could open an issue on the Python extension repo and see if they're open to it? If you do, can you link it here?

**Reply by Msadat97** [reply]
I opened an issue requesting this feature here: https://github.com/cdr/code-server/issues/4467

**Reply by jsjoeio** [reply]
Ah, yes you should open it under [this repo](https://github.com/Microsoft/vscode-python/issues) instead
